### PR TITLE
bug fix: deb12 support (#12)

### PR DESCRIPTION
**Pull request by nishanthsenthilvasagam**
Kindly verify and merge

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,11 +1,14 @@
 sudo apt update
 sudo apt install astyle cmake gcc ninja-build libssl-dev python3-pytest python3-pytest-xdist unzip xsltproc doxygen graphviz python3-yaml python3-pip git
-pip3 install cryptography
-pip3 install argon2-cffi
+sudo apt install python3.11-venv
```
Comment (Anish-M-code) [inline install.sh]: I guess this can be sudo apt install python3-venv , we should avoid fixing it to 3.11

**Comment by Anish-M-code** [comment]
Also since we are modifying install.sh we should also add corresponding changes to run.sh which is used for starting pqcrypt.

**Comment by nishanthsenthilvasagam** [comment]
I have made the changes. Kindly do the needful.

**Review by Anish-M-code — APPROVED** [review]

**Comment by Anish-M-code** [comment]
thank you for contributing @nishanthsenthilvasagam

**Pull request merged**